### PR TITLE
Adds arms dealer, weapon shelf, gunshop

### PR DESCRIPTION
**Pull request by Salex08**
## About The Pull Request

Request from happyman1997, we also have Bones blessing
grafik

## Why It's Good For The Game

New Job which sells guns, armor. New shelfs which allow to store weaponry. Now there are Gunshops in every map

## Changelog
:cl:
add: new occupation Arms Dealer
add: new shelfs for storing weaponry
/:cl:

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
Make the vending machine only accessible to that role. No captain or HOS either, JUST that role.

**Comment by Salex08** [comment]
> Make the vending machine only accessible to that role. No captain or HOS either, JUST that role.

done

**Comment by karmaisblackandbluepilled** [comment]
Ok good. I wanted this change because otherwise all the retards would instantly rush the shop for weapons.
Gonna ask @EtraIV or @QuakeIV to take a look at this too.

**Comment by karmaisblackandbluepilled** [comment]
Oh yeah I just noticed, you only added the shop to metastation? But the role can be picked on every map.

**Comment by Salex08** [comment]
> Talked to other jannies, rename it to arms dealer or something like that, gunsman is weird and usually means mass shooter rather than guy who sells weapons.

I dont know why you guys think that, but here I renamed it

**Comment by karmaisblackandbluepilled** [comment]
merging this lol

**Comment by karmaisblackandbluepilled** [comment]
actually lemme review this first

File: code/game/area/space_station_13_areas.dm [hunk 1]
```diff
@@ -712,7 +712,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 
 /area/service/theater
 	name = "Theater"
-	icon_state = "Theater"
+	icon_state = "theatre"
```
[the remark this answers is not in the document]
Comment (Salex08) [reply code/game/area/space_station_13_areas.dm:715]: thats a bug the icons name is theatre not theater

File: code/game/objects/structures/shelf.dm [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+	stored += list(
+		new /obj/item/gun/ballistic/rifle/boltaction,
+		new /obj/item/gun/ballistic/rifle/boltaction/brand_new,
+		new /obj/item/gun/ballistic/rifle/boltaction/harpoon,
+		new /obj/item/gun/ballistic/rifle/boltaction/pipegun,
+		new /obj/item/gun/ballistic/rifle/boltaction/pipegun/prime,
+	)
+
+/obj/structure/shelf/ammo
+	name = "Ammoshelf"
```
[the remark this answers is not in the document]
Comment (Salex08) [reply code/game/objects/structures/shelf.dm:163]: ok ok

[the remark this answers is not in the document]
Comment (Salex08) [reply code/game/objects/structures/shelf.dm]: done

**Comment by karmaisblackandbluepilled** [comment]
etra is AWOL so  I am just gonna merge this

**Comment by Salex08** [comment]
> etra is AWOL so I am just gonna merge this

based

**Pull request merged**